scene/register_scene_types.cpp

```diff
@@ -511,6 +512,7 @@ void register_scene_types() {
 	GDREGISTER_CLASS(GPUParticlesAttractorVectorField3D);
 	GDREGISTER_CLASS(CPUParticles3D);
 	GDREGISTER_CLASS(Position3D);
+	GDREGISTER_CLASS(AnimatedSprite3D);
```

Comment:
This is already registered, was that a wrongly resolved merge conflict?

Reply:
Oops. Must've accidentally added that. Fixed now. :)